conftest.py

```diff
@@ -92,6 +93,13 @@ def pytest_addoption(parser: Parser) -> None:
         help="Delete pre-upgrade resources; useful when debugging pre-upgrade tests",
     )
 
+    # Upstream options
+    upstream_group.addoption(
+        "--model-registry-upstream",
+        default=False,
```

Comment:
use `action="store_true",` so the user does not have to pass `=True` but only pass `--model-registry-upstream`

Reply:
Done